.github/workflows/spec_zero.yml

```diff
@@ -57,6 +57,6 @@ jobs:
           git checkout -b spec_zero
           git commit -am "MAINT: Update dependency specifiers"
           git push origin spec_zero
-          PR_NUM=$(gh pr create --base main --head spec_zero --title "MAINT: Update dependency specifiers" --body "Created by spec_zero [GitHub action](https://github.com/mne-tools/mne-python/actions/runs/${{ github.run_id }})." --label "no-changelog-entry-needed")
+          PR_NUM=$(gh pr create --base main --head spec_zero --title "MAINT: Update dependency specifiers" --body "Created by spec_zero [GitHub action](https://github.com/mne-tools/mne-python/actions/runs/${{ github.run_id }}).\n\n*Adjustments may need to be made to shims in mne/fixes.py in this or another PR. `git grep TODO VERSION` is a good starting point for finding potential updates.*" --label "no-changelog-entry-needed")
```

Comment:
@drammock @tsbinns I pushed a commit here -- seeing this in practice, I think it's worth saying in the body of the PR that some manual intervention is needed to complete the work. I added a note for `credit.yml` as well, and pushed the manual work to this PR already.